### ISSUE TITLE
Laboratory prototype created with DX cooling

### ISSUE THREAD

**Issue by eringold** [NREL/openstudio-standards#1277]
Laboratory prototype model created in CZ 3B is created with Coil:Cooling:DX:TwoSpeed coils on both AHUs. 

The best source I could find on the development of this model came from this I2SL presentation by Amy Allen: https://www.i2sl.org/conference/2019/presentations/c2_allen.pdf

which specifies Chiller as the cooling source for both: 
![image](https://user-images.githubusercontent.com/10799524/157322860-a96b5b8c-d590-4e48-a1b0-416e36461668.png)

**Comment by KaiyuSun** [comment]
Water-cooled chiller was our initial assumption on the cooling source. After getting feedback from TC9.10 and 90.1 committee, we decided that water-cooled chiller is not appropriate for lab building at this size. It was updated to packaged VAV based on Appendix G baseline system types and suggestion from the above group. 

The latest assumptions are documented in the scorecard, which can be found [here](https://github.com/NREL/openstudio-standards/tree/master/docs/scorecards).
